diesel/src/connection/sqlite/mod.rs

```diff
@@ -0,0 +1,138 @@
+extern crate libsqlite3_sys as ffi;
```

Comment:
Glob importing the crate causes namespace conflicts, and it felt like the most appropriate name to use as a namespace.